README.md

```diff
@@ -1,2 +1,185 @@
-# action
-GitHub Action for code bases using Git Town's stashed commits feature
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/git-town/git-town/main/website/src/logo.svg">
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/git-town/git-town/main/website/src/logo-dark.svg">
+    <img alt="Git Town logo" src="https://raw.githubusercontent.com/git-town/git-town/main/website/src/logo.svg">
+  </picture>
+</p>
+
+# Git Town Action V1
+
+This action visualizes your stacked changes when proposing pull requests on GitHub:
+
+- `main`
+  - ["git town park" command #3181](https://github.com/git-town/git-town/pull/3181) :point_left:
+    - ["git town observe" command" #3186](https://github.com/git-town/git-town/pull/3186)
+
+This allows you to easily see all related PRs for a given pull request, where
+you are in the stack, as well as navigate between PRs in a stack.
+
+It is designed to work out of the box with [Git Town](https://github.com/git-town/git-town) v12+, but also supports previous versions via the `perennial-branches` & `perennial-regex` inputs.
+
+# What's New
+
+Please refer to the [release page](https://github.com/git-town/action/releases/latest) for the latest release notes.
+
+# Usage
```

Comment:
Sure!

Reply:
Sorry for being unclear. My point is that the "Usage" and "Getting Started" sections contain content that belongs together. Because to use this action, one needs to configure it. So configuring this action is a part of setting it up. Hence there should be only one section that explains how to set up and configure this action, not two. 

The end result could look something like this:

```md
## Installation

### Create the GitHub Actions workflow file

### Create the configuration file for this action

### Configuring perennial branches
```

Reply:
I'm following you now! Let me get that fixed up real quick!